core/server/data/exporter/table-lists.js

```diff
@@ -50,6 +50,7 @@ const TABLES_ALLOWLIST = [
     'roles',
     'roles_users',
     'settings',
+    'custom_theme_settings',
```

Comment:
Added to the general export tables because it felt like custom theme settings are an extension of site settings. The active theme is exported as part of site settings so having your customisations of that theme exported too felt like a reasonable expectation. Happy to revise to only include in the full "backup" export if that better fits the intention of export/backup split.